generator.js

```diff
@@ -1,23 +1,15 @@
+const { pluginVersion } = require('./version')
+
 module.exports = (api, options, rootOptions) => {
   api.extendPackage({
     scripts: {
       deploy: `vue-cli-service s3-deploy`
     }
   })
 
-  // Set some defaults
   // Override these in a .env file or in vue.config.js
-  options.awsProfile = 'default'
-  options.region = 'us-west-1'
-  options.createBucket = false
-  options.staticHosting = false
-  options.assetPath = 'dist'
-  options.assetMatch = '**'
-  options.deployPath = '/'
-  options.acl = 'public-read'
-  options.pwa = false
-  options.enableCloudfront = false
```

Comment:
@multiplegeorges Why were all these removed? They were really useful since I use a preset instead of invoking this plugin.